src/ui/utils/browser.ts

```diff
@@ -1,15 +1,3 @@
-export function setUserInBrowserPrefs(user: { sub: string } | null) {
```

Comment:
This isn't actually supported in our browser anymore (I get "no such channel" errors when using it) so I removed it completely.